examles/example1.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>svg2pdf - save file as pdf</title>
+  </head>
+  <body>
+    <button id="downloadpdf">Download PDF</button>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.0.0/jspdf.umd.min.js" integrity="sha512-g77bZKU4ktH2I5nNioWzOMcbd3fN/svB0vQM73Uo5GRn/EGfcSJB0DlR1ithxnFsDaa0HmlOwHYiUFeM1P3RRw==" crossorigin="anonymous"></script>
+    <script src="../dist/svg2pdf.umd.min.js"></script>
+
+    <script>
+      const { jsPDF } = window.jspdf
+
+      var svgText = '<svg width="100" height="100"><rect x="0" y="0" width="200" height="100" fill="green"></rect></svg>'
```

Comment:
That's a weird SVG, only showing a rectangle that is larger than the viewport. Couldn't we use something more beautiful? This is test-case image quality.

Reply:
Sure, I will modify the svg image. My idea was to give an example, How to use svg2pdf lib to allow user to download an image which generated sricpt?